### PR TITLE
Take another look at our JS coding standards

### PR DESCRIPTION
**Pull request by thatdevgirl**
Trello card: https://trello.com/c/8jJfqDZS

### CODE REVIEW

File: codeguide.md [hunk 1]
```diff
@@ -290,51 +290,44 @@ Include Sass/CSS linting in the [Atom editor](https://atom.io/) by ensuring that
...
+* Functionality should be developed inside a `source/` directory. The compiled file should be built into a `build/` directory. There should be a single built file for any theme or plugin.
+  * Theme development files: `themes/THEME_NAME/source/js`
+  * Theme built file: `themes/THEME_NAME/build/js`
+  * Plugin development files: `/plugins/PLUGIN_NAME/source/js`
+  * Plugin built file: `plugins/PLUGIN_NAME/build`
 
-* __Plugins:__ Functionality that is global to our WordPress environment and not specific to a theme should reside in its own plugin. The file structure of a given plugin may differ slightly based on the complexity of the theme, but as a general rule, the Javascript files should reside in `plugins/PLUGIN_NAME/COMPONENT_NAME/source`. Compiled and minified Javascript will reside in `plugins/PLUGIN_NAME/COMPONENT_NAME/build`.
+* The structure of any Javascript files within a theme or plugin should adhere to the following patter, but may slightly differ based on the complexity of that theme or plugin.
```
Comment (reedcodes) [inline codeguide.md]: Typo for "pattern"

Comment (dealancer) [reply codeguide.md]: I hear the rhythm here: "patter" and "differ" :) So may be OK? (kidding)

File: codeguide.md [hunk 2]
```diff
@@ -349,14 +342,35 @@ $( '#foo' ).click( () => {
...
 
-* Put the beginning bracket at the end of the first line of the block.
-* Use one space before the opening bracket.
-* Indent each line inside the code block two spaces.
-* Put the ending bracket on the last line, with no preceding spaces.
-* Do not include spaces before or after parameters or expressions inside parenthesis.
-* Break one line for statements in the same code block. Break two lines before and after code blocks.
+* Blocks should always use braces (`{ }`) and always go on multiple lines.
+  * The beginning brace should be at the end of the first line of the block.
+  * The ending brace should be on the last line, with no preceding spaces.
```
Comment (dealancer) [inline codeguide.md]: With no preceding spaces? What about indents of the block itself?

File: codeguide.md (hunk 1 above)
Comment (thatdevgirl) [reply codeguide.md]: It's raining in our JS file structure.

(This will be fixed with my next commit.)

File: codeguide.md (hunk 2 above)
Comment (thatdevgirl) [reply codeguide.md]: Good point! Not what I mean; I will clarify.

**Comment by reedcodes** [comment]
[Here is a sample card that is moving through the sprint.](https://trello.com/c/pDVQEc5j/18-here-is-a-sample-card-that-is-moving-through-the-sprint)

**Pull request merged**